### DISCUSSION TITLE
menu not updating for Board part number

### DISCUSSION THREAD

**Discussion by emoninet2** [stm32duino/Arduino_Core_STM32#2029] [Q&A] [answered]
I have followed all the steps provided in [https://github.com/stm32duino/Arduino_Core_STM32/wiki/Add-a-new-variant-%28board%29](https://github.com/stm32duino/Arduino_Core_STM32/wiki/Add-a-new-variant-%28board%29) 
to add the new variant. However, on the Arduino IDE under **tool>Board part number** , I cannot see the newly added generic part. What am I missing?

**Comment by fpistm** [comment, marked as answer]
Hi @emoninet2 
If you use Arduino IDE 2, it is a know problem, the boards.txt is no more parsed. So you have to delete all files from dedicated folders. on Windows it is: C:\Users\<name>\AppData\Roaming\arduino-ide

**Reply by emoninet2** [reply]
Thanks @fpistm . That worked. :-)
